docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  remacs:
+    build: .
+    image: remacs-build:latest
+    working_dir: ${PWD}
+    tty: true
+    # Unfortunately dumping doesn't work without this
```

Comment:
Presumably this is due to: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=23529 (see also [this](https://emacs.stackexchange.com/a/22667/304)).